espflash/src/bin/espflash.rs

```diff
@@ -156,6 +204,36 @@ fn main() -> Result<()> {
     }
 }
 
+fn erase_flash(args: EraseFlashArgs, config: &Config) -> Result<()> {
+    info!("Erasing Flash...");
+    let mut flash = connect(&args.connect_args, config)?;
+    flash.erase_flash()?;
+
```

Comment:
Is there any reason not to `flash.connection().reset()` here as we do in `erase_parts` and `erase_region`?

Reply:
I do have a `reset()` in place after `erase_flash`, but it's in `flasher/mod.rs` instead of here.  Your comment here makes me realize that's inconsistent with the other operations.  I'll move this into `espflash.rs` as suggested.